kuma/settings/common.py

```diff
@@ -1226,6 +1224,20 @@ def pipeline_one_scss(slug, **kwargs):
     cast=bool
 )
 
+# Allow robots, but restrict some paths
+# If the domain is a CDN, the CDN origin should be included.
+ALLOW_ROBOTS_WEB_DOMAINS = set(
+    config('ALLOW_ROBOTS_WEB_DOMAINS',
+           default='developer.mozilla.org',
+           cast=Csv()))
+
+# Allow robots, no path restrictions
+# If the domain is a CDN, the CDN origin should be included.
+ALLOW_ROBOTS_DOMAINS = set(
```

Comment:
I think this name should be `ALLOW_ROBOT_MEDIA_DOMAINS`

Reply:
I've been thinking about ``ALLOW_ROBOT_MEDIA_DOMAINS``. ``MEDIA`` suggests Django's media upload directory to me, but this is the domain for:

* attachments - Files uploaded by users, close to the Django media upload. Eventually I want to move these to their own domain backed by S3 or a new file upload service, and get Django out of the loop.
* code samples - These are extracted from the content, repackaged as HTML, and served in an ``<iframe>``. Again, the implementation may change in the future, and maybe even Django won't be involved in serving the content.

Together, we call these the samples domain, or the attachments domain, or the untrusted domain. The fact that we can't settle on a name is a good indication that the design is wrong.

So, I'm not going to further confuse naming by adding ``MEDIA`` in there. It's ``ALLOW_ROBOTS_DOMAINS``, where robots are allowed without restrictions.

Reply:
I think the defaults should be the config defaults for these:

```
- ALLOW_ROBOTS_DOMAINS=${ALLOW_ROBOTS_DOMAINS:-mdn.mozillademos.org,mdn-demos-origin.moz.works}
- ALLOW_ROBOTS_WEB_DOMAINS=${ALLOW_ROBOTS_WEB_DOMAINS:-developer.mozilla.org}
```

That way a user can override it if they are working on ``robots.txt``, but leave it in "no indexing please" mode if they aren't. This makes be think that all the environment overrides should be in ``docker-compose.yml``, with notes for the non-default ones. Which makes me think our settings need fixing. Which makes me lazy and want to do it in a new PR.